packages/forms/backend/sirius-components-collaborative-forms/src/main/java/org/eclipse/sirius/components/collaborative/forms/api/DiagramFilterConfiguration.java

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.collaborative.forms.api;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+
+import org.eclipse.sirius.components.collaborative.api.IRepresentationConfiguration;
+
+/**
+ * The configuration used to create a diagram filter event processor.
+ *
+ * @author gdaniel
+ */
+public class DiagramFilterConfiguration implements IRepresentationConfiguration {
+
```

Comment:
There is problem if several users display the new filter elements menu on the same diagram at the same time.
The selection of diagram elements are synchronized but it should not be the case.
If user A select some checkboxes in his menu, user B should not view the checkboxes selected in its own menu.
the only thing that should be synchronized are the icons located after the checkboxes labels, indicating the states (hide,show,fade...) of diagram elements.

This is probably due to the fact that everybody shares the same formId here.
The frontend should probably use a random UUID when the subscription is called.

Reply:
I fixed it in `DiagramFilterConfiguration`, which now computes the form ID from the received object id and UUID passed by the frontend (paremeters `id` and `objectIds` in `PropertiesEventInput`).